docs/search_syntax.md

```diff
@@ -0,0 +1,281 @@
+  - [Logical operations](#logical-operations)
+- [Search run examples](#search-run-examples)
+  - [Example with run.name (string)](#example-with-runname-string)
+  - [Example with run.duration (numeric)](#example-with-runduration-numeric)
+  - [Example with run.archived (boolean)](#example-with-runarchived-boolean)
+  - [Run parameters](#run-parameters)
+  - [Filtering Runs with Unset Attributes](#filtering-runs-with-unset-attributes)
+  - [Filter Runs using Regular Expressions](#filter-runs-using-regular-expressions)
+  - [Complex query for run search](#complex-query-for-run-search)
+- [Search metrics examples](#search-metrics-examples)
+  - [Example with metric.name (string)](#example-with-metricname-string)
+  - [Example with metric.last (numeric)](#example-with-metriclast-numeric)
+  - [Filter Metrics by run](#filter-metrics-by-run)
+  - [Complex query for metric search](#complex-query-for-metric-search)
+
+
+## Search Runs
+You can filter the runs using the following run attributes:
+| Property           | Description                                         | Type             |
+| ------------------ | --------------------------------------------------- | ---------------- |
+| ```name```         | Run name                                            | ```string```     |
+| ```hash```         | Run hash                                            | ```string```     |
+| ```experiment```   | Experiment name                                     | ```string```     |
+| ```tags```         | List of run tags                                    | ```dictionary``` |
+| ```archived```     | True if run is archived, otherwise False            | ```boolean```    |
+| ```active```       | True if run is active(in progress), otherwise False | ```boolean```    |
+| ```duration```     | Run duration in seconds                             | ```numeric```    |
+| ```created_at```   | Run creation datetime                               | ```numeric```    |
+| ```finalized_at``` | Run end datetime                                    | ```numeric```    |
+| ```metrics```      | Set of run metrics                                  | ```dictionary``` |
+
+## Search Metrics
+You can filter the metrics using the following metric attributes:
+| Property         | Type          |
+| ---------------- | ------------- |
+| ```name```       | ```string```  |
+| ```last```       | ```numeric``` |
+| ```last_step```  | ```numeric``` |
+| ```first_step``` | ```numeric``` |
+
+## Operations
+
+### String operations
+For the ```string``` attributes you can use the following comparing operator:
+- ``` == ```
+- ``` != ```
+- ``` in ```
+- ``` .startswith() ```
+- ``` .endswith() ```
+
+### Numeric operations
+For the ```numeric``` attributes you can use the following comparing operator:
+- ``` == ```
+- ``` != ```
+- ``` > ```
+- ``` >= ```
+- ``` < ```
+- ``` <= ```
+
+### Boolean operations
+For the ```boolean``` attributes you can use the following comparing operator:
```

Comment:
Please also document that having no operator or a unary operation also works for booleans, e.g. `run.archived` is the same as `run.archived == True`, and `!run.archived` is the same as `run.archived == False`